test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml

```diff
@@ -16,15 +16,15 @@
   ~ limitations under the License.
   ~
   -->
-<Configuration status="WARN">
+<Configuration status="info">
     <Appenders>
-        <Console name="Console" target="SYSTEM_ERR">
-            <PatternLayout charset="UTF-8" pattern="[%d{yyyy-MM-dd HH:mm:ss:SSS}] [%p] - %l - %m%n"/>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d [%traceId] %-5p %c{1}:%L - %m%n"/>
         </Console>
     </Appenders>
     <Loggers>
-        <Root level="WARN">
+        <Root level="info">
```

Comment:
I think it is higher level better. this level will print many messages.

Reply:
done